test/issues/github_304.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright Jason Rice 2016
```

Comment:
Can we please rename this file  to `test/ext/std/array/issue_304.cpp`? This way, the dependency upon external adapters is explicit.